src/components/CreatePost/PostBasic.tsx

```diff
@@ -10,6 +10,11 @@ const PostBasic = () => {
   // api 연결시 커스텀 훅으로 분리 예정
   const [startDate, setStartDate] = useState<Dayjs | null>(dayjs(''));
```

Comment:
proposal;

```suggestion
 // 타입 별칭 이름은 변경해야겠네요...
  type UseDayjsType = Dayjs | null;
  const [startDate, setStartDate] = useState<UseDayjsType>(dayjs(''));
```

<Dayjs | null> 타입이 여러 곳에서 사용되니 타입 별칭으로 뽑아서 사용하면 중복을 제거할 수 있어요.

Reply:
해당부분은 react-hook-form 을 사용하면서 모두 제거되었습니다!